satpy/tests/test_composites.py

```diff
@@ -129,7 +129,7 @@ def test_nondimensional_coords(self):
         self.assertNotIn('acq_time', ret_datasets[0].coords)
 
 
-class TestRatioSharpenedCompositors(unittest.TestCase):
+class TestRatioSharpenedCompositors:
     """Test RatioSharpenedRGB and SelfSharpendRGB compositors."""
 
     def setUp(self):
```

Comment:
Change this to `def setup_method` and add the `self.ds1 =`-style lines back. Unless this data is really only used by one test in this class. If so, then it should be renamed to something like `_create_fake_data`.